config/locales/en.yml

```diff
@@ -21,3 +21,4 @@
 
 en:
   hello: "Hello world"
+  badge: "Crachá"
```

Comment:
This should be under `pt` 😞 but I'll allow it since it's not the time to handle locale settings right now.